setup/install/providers/aws/aws-ec2.md

```diff
@@ -11,13 +11,19 @@ sidebar:
 > worked on. In the meantime, please use the following
 > [AWS tutorial: Continuous Delivery using Spinnaker on Amazon EKS](https://aws.amazon.com/blogs/opensource/continuous-delivery-spinnaker-amazon-eks/).
 
-Use the AWS EC2 Provider if you want to manage EC2 Instances via Spinnaker. Refer to the [AWS Cloud Provider Overview](https://spinnaker.io/setup/install/providers/aws/) to understand how AWS IAM must be set up for the AWS EC2 provider to work.
+Use the AWS EC2 Provider if you want to manage EC2 Instances via Spinnaker. Refer to the [AWS Cloud Provider Overview](https://spinnaker.io/setup/install/providers/aws/) to understand how AWS IAM must be set up with the Spinnaker AWS EC2 provider.
+
+Spinnaker will use an [AWS IAM structure](https://aws.amazon.com/iam/) of users, roles, policies, etc to access AWS services and resources securely. There are 3 options to set up the AWS IAM structure
+
+1. AWS CloudFormation templates deployed with the CloudFormation Console
```

Comment:
Maybe hyperlink these to their respective headings below?

Reply:
I meant link each of these three list items to the headings below, in the article (though with headings like these, the auto-generated anchor will be complex. The current anchor for option 1 (on the published site, before these changes) is `#option-1--use-aws-console-to-configure-aws`